ziggurat-core-crawler/src/connection.rs

```diff
@@ -0,0 +1,54 @@
+use std::{
+    cmp::Ordering,
+    hash::{Hash, Hasher},
+    net::SocketAddr,
+    time::Instant,
+};
+
+/// A connection found in the network.
+#[derive(Debug, Eq, Copy, Clone)]
+pub struct KnownConnection {
+    /// One of the two sides of a connection.
+    pub a: SocketAddr,
+    /// The other side of a connection.
+    pub b: SocketAddr,
+    /// The timestamp of the last time the connection was seen.
+    pub last_seen: Instant,
```

Comment:
Just wondering whether there should be `Instant` or `SystemTime` used.
As the first one can be used only to measure the duration between two instants I wonder if using `SystemTime` would not be better to get some specific date/time (we can easily get human-readable last time seen).

Reply:
`SystemTime` is not monotonic and so a bit unreliable for the things we use it for. Right now `Instant` is used across ziggurats, so I would suggest leaving it and maybe decide later after checking how big of an impact it would be.